### DISCUSSION TITLE
https://e107.org/ --> Down

### DISCUSSION THREAD

**Discussion by Alex-e107nl** [e107inc/e107#5154] [General]
There's no e107.org.... serverproblems? My newsfeedreader on e107.nl timed out...

Record update dates
Created date: December 16, 2023 09:00:25 UTC
Updated date: December 16, 2023 09:00:25 UTC

**Comment by tgtje** [comment]
12.20 running

**Comment by Deltik** [comment]
I'm seeing evidence of a UDP-based denial-of-service attack that ran from about 06:40 UTC to 11:00 UTC on 16 December 2023.  There also seems to have been a less successful TCP attack from 11:35 UTC to 12:00 UTC.

I'll chat with @CaMer0n to explore attack mitigation strategies.
